app/views/dashboard/manageAgents.scala.html

```diff
@@ -0,0 +1,41 @@
+@import controllers.agent.RepresentationController.ManagePropertiesVM
+
+@(model: ManageAgentsVM)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    Messages("dashboard.manageAgents.title"), topNavigation = true,
+    topBackLink = true, bottomBackLink = true
+) {
+    <h1 class="heading-xlarge margin-bottom-0 clear-right">
```

Comment:
I'm not sure about this h1 block here, though I know it's used on multiple pages, the span and different classes inside might not fit within recognised patterns for the platform. http://hmrc.github.io/assets-frontend/section-header.html

Ideally I think it would look like
```
<header class="page-header">
<h1>@Messages("dashboard.manageAgents.title")</h1>
@if(model.agents.isEmpty) {
   <h2 id="noAgents">@Messages("dashboard.manageAgents.noAgents")</h2> 
}
</header>
```